workers/agent/agent.go

```diff
@@ -454,9 +455,76 @@ func (a *Agent) scheduleWork() {
 
 		// Let us try to start the plugin and receive all implemented jobs
 		if err = a.scheduler.SetPipelineJobs(pipeline); err != nil {
-			gaia.Cfg.Logger.Error("cannot get pipeline jobs", "error", err.Error(), "pipelinerun", pipelineRunPB)
-			reschedulePipeline()
-			return
+			if strings.Contains(err.Error(), "exec format error") {
```

Comment:
It makes the code more readable if you do a negative check beforehand. e.g.:
```
if !strings.Contains(err.Error(), "exec format error") {
  gaia.Cfg.Logger.Error("cannot get pipeline jobs", "error", err.Error(), "pipelinerun", pipelineRunPB)
   reschedulePipeline()
   return
}
```

Reply:
I don't intend to actually keep this in this way. :D